CONTRIBUTING.md

```diff
@@ -125,26 +131,26 @@ a feature request use the [Feature
 Request](https://github.com/osquery/osquery/issues/new?template=Feature_Request.md) template.
 
 **Please only use issues for bug reports or feature requests**. If you have deployment questions or
-issues or a general question about osquery hit our Slack instead as you'll have better support
+issues or a general question about osquery hit our Slack instead as you will have better support
```

Comment:
```suggestion
issues or a general question about osquery use our Slack instead as you will have better support
```

Of course, I always want to revisit the stackoverflow stuff. But maybe kick that forward into some larger addressing